sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -310,6 +310,11 @@ static struct snd_soc_acpi_codecs tgl_max98373_amp = {
 	.codecs = {"MX98373"}
 };
 
+static struct snd_soc_acpi_codecs tgl_rt1011_amp = {
+	.num_codecs = 1,
+	.codecs = {"10EC1011"}
+};
+
```

Comment:
wondering if these tables can be const? I know all other tables are not, but I see no reason why.

Reply:
i think it's ok to make it const since .quirk_data is only accessed by snd_soc_acpi_codec_list() without modifying it.

Reply:
the important point here is, that `.quirk_data` is declared `const` in `struct snd_soc_acpi_mach`:
```
struct snd_soc_acpi_mach {
	...
	const void *quirk_data;
```
so, yes, it's good.